src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -1078,6 +1078,7 @@ void network::build_insts_deps() {
     GPU_DEBUG_DEFINE_MEM_LOGGER("build_insts_deps");
     for (auto& inst : _primitives) {
         inst.second->build_deps();
+        inst.second->configure_shape_of_dependencies();
```

Comment:
Yeah, it can be, but some instances call build_deps() from on_execute() method, so I added independent function to configure shape_of dependency only once